docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -13,11 +13,39 @@ The main features of the Media Controller API include:
 
   When the client requests the information, the media controller server updates the state information of an active application before transferring the data. If the application is not running when the client request arrives, the media controller server transfers the latest information.
 
+- Update and retrieving playlist
+
+  You can [update the playlist information](#get_playlist) on the server side, and then retrieve the playlist information on the client side.
+
+  The media controller server provides current information about the registered application that you can send to the client.
+
+  **Note**
+
+  This feature supports Tizen 4.0 and higher for Mobile.
+
 - Sending and processing commands
 
-  You can [send a command](#send_media) to the server from the client side, and then process the command on the server side.
+  You can [send a command](#send_command) to the server from the client side, and then process the command on the server side.
+
+- Sending and processing commands to receive replies
+
+  You can [send a command](#send_cmd) to the server from the client side, and then process the command on the server side.
+
+  You can [send a reply of the command](#send_cmd_reply) to the client from the server side, and then receive the reply on the client side.
 
-  The client can request [server state](#serverstate) or [server metadata](#servermetadata) information from the server, and receive it through a callback.
+  **Note**
+
+  This feature supports Tizen 4.0 and higher for Mobile.
+
+- Sending and processing a custom event
+
+  You can [send a custom event](#send_event) to the client from the server side, and then process the event on the client side.
+
+  You can [send a reply of the custom event](#send_event_reply) to the server from the client side, and then receive the reply on the server side.
+
+  **Note**
```

Comment:
From line 46 to line 48, please add "> " in front of the lines.
```
   > **Note**
   >
   > This feature supports Tizen 4.0 and higher for Mobile.
```

Please make sure there are 3 spaces in front of ">".

Reply:
I added "> " in front of the Note.